sql/load_demo_data.sql

```diff
@@ -6100,7 +6290,11 @@ CREATE TABLE `oh_session_audit` (
   `SEA_ACTIVE` tinyint(1) NOT NULL DEFAULT 1,
   PRIMARY KEY (`SEA_ID`),
   KEY `SEA_US_ID_A` (`SEA_US_ID_A`),
-  CONSTRAINT `oh_session_audit_ibfk_1` FOREIGN KEY (`SEA_US_ID_A`) REFERENCES `oh_user` (`US_ID_A`)
+  KEY `FK_SESSION_AUDIT_CREATED_BY_IDX` (`SEA_CREATED_BY`),
+  KEY `FK_SESSION_AUDIT_LAST_MODIFIED_BY_IDX` (`SEA_LAST_MODIFIED_BY`),
+  CONSTRAINT `FK_SESSION_AUDIT_CREATED_BY` FOREIGN KEY (`SEA_CREATED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `FK_SESSION_AUDIT_LAST_MODIFIED_BY` FOREIGN KEY (`SEA_LAST_MODIFIED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `oh_session_audit_ibfk_1` FOREIGN KEY (`SEA_US_ID_A`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
The sql 04 version doesn't have the `ON DELETE CASCADE ON UPDATE CASCADE` part at the end of line 6297
![image](https://github.com/informatici/openhospital-core/assets/1105445/0c44f872-b247-47f5-ae61-64ecb227487f)

Reply:
that's why there was that part in step_95 redefining it... I try to read it and change the load_demo_data.sql
thanks